### PR TITLE
Deprecated library fixes

### LINKED ISSUE #854: Avoid deprecated library
When the app runs in the dev mode (:3000), we get this warning:
image

Here's the link: https://developer.chrome.com/blog/enabling-shared-array-buffer/

I don't really understand it, but am concerned something may break next month.

We should investigate the issue, and see if it's in our code (and we fix it), or an upstream dependancy (so we update dependency).

### PR DESCRIPTION
**Pull request by sanjayradadiya**
## 🧰 Issue

Fixes #854 

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

## 📝 Developer Notes:

Please review this thread
https://github.com/serge-web/serge/pull/871/files#r658745999

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -51,13 +51,13 @@
...
     "@typescript-eslint/eslint-plugin": "^2.20.0",
     "@typescript-eslint/parser": "^2.20.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
     "babel-loader": "^8.0.6",
     "classnames": "^2.2.6",
     "copyfiles": "^2.2.0",
     "css-loader": "^3.4.2",
     "css-modules-typescript-loader": "^4.0.0",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
```
Comment (sanjayradadiya) [inline packages/components/package.json:60]: test cases was breaking due to react-16 adapter so we needed adapter for react version 17 and that is not available in market as this is perfectly new version.

so i just used this suggestion and added temporary package which will solve our issues after upgrades and later on whenever exact package release officially then we just need to update it.

https://stackoverflow.com/a/64597067/6536418

Comment (IanMayo) [reply packages/components/package.json:60]: Thanks @sanjayradadiya 

Could you add a comment in `package.json` next to the dependency, including a link to the Stack Overflow issue please?  Could you also create a GitHub issue in this repo explaining that we need to switch back to the proper adapter?

Comment (IanMayo) [reply packages/components/package.json:60]: Hello @sanjayradadiya - I'm keeping this PR open as a reminder that we need the above comment/documentation sorted.

Once you've done it I can merge the PR :-)

Comment (IanMayo) [reply packages/components/package.json:60]: This PR still needs some attention from you, please @sanjayradadiya

Comment (sanjayradadiya) [reply packages/components/package.json:60]: Ahh, okay

Comment (sanjayradadiya) [reply packages/components/package.json:60]: Please check this issue documentation here for sure

https://github.com/serge-web/serge/issues/950

Comment (IanMayo) [reply packages/components/package.json:60]: Nicely done, thanks @sanjayradadiya

**Review by IanMayo — DISMISSED** [review]

**Comment by IanMayo** [comment]
Close this. Other PRs have resolved these issues.

**Pull request closed without merging**